persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MolecularDataMapper.xml

```diff
@@ -62,6 +63,36 @@
         </where>
     </select>
 
+    <!-- This routine is a copy of getGeneMolecularAlterations above.  This copy is necessary because
```

Comment:
is there a way that you can move all of this into a `<sql> .... </sql>` element that is referenced by these queries using `include` as it is done [here](https://github.com/cBioPortal/cbioportal/pull/6457/files#diff-4020171d42dc9dc6eb6f0c4e437a48a0R110)

Reply:
@ao508 this is not from this pr.  It's introduced in https://github.com/cBioPortal/cbioportal/commit/685912300513b9b92512c54c9dfbb84080792411 I will rebase once the latest master is in rc branch.